homeassistant/components/vicare/__init__.py

```diff
@@ -53,7 +55,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     hass.data[DOMAIN] = {}
     hass.data[DOMAIN][entry.entry_id] = {}
 
-    await hass.async_add_executor_job(setup_vicare_api, hass, entry)
+    try:
+        await hass.async_add_executor_job(setup_vicare_api, hass, entry)
+    except PyViCareInvalidCredentialsError as ex:
+        raise ConfigEntryAuthFailed("Authentication failed") from ex
```

Comment:
Is there a way to use the localized `invalid_auth` text here?

Reply:
Not yet. There is a PR open to start implementing translatable errors, but it still needs to be merged. We can add translations for it at a later moment.